README.adoc

```diff
@@ -71,27 +56,52 @@ different command line options for `docker-compose`, consult the
 https://docs.docker.com/compose/overview/[official docker-compose documentation].
 
 === Getting Started
-
 IMPORTANT: SMART COSMOS DevKit is designed for learning, testing, and evaluation
 of the SMART COSMOS platform. It is not intended for production use, and it is
-inherently insecure due to its reliance on a JKS key pair for which the
-https://github.com/SMARTRACTECHNOLOGY/smartcosmos-auth-server/blob/master/src/main/resources/smartcosmos.jks[secret key]
+inherently insecure due to its reliance on a JKS key pair for which the https://github.com/SMARTRACTECHNOLOGY/smartcosmos-auth-server/blob/master/src/main/resources/smartcosmos.jks[secret key]
 is publicly available and cannot be replaced, and its use of the primary Objects
 database as a store for user and account data. To establish a secure, production-ready
 SMART COSMOS environment, consult your SMARTRAC Technology Inc. sales representative.
 
 To jump right into operating the DevKit, follow our
-link:tutorials/getting-started.adoc[getting started tutorials].
+link:guides/getting-started.adoc[getting started tutorials].
 
-[[nextSteps]]
-== Next Steps
+== Open Source Framework
+SMART COSMOS includes a number of open source frameworks available under the
+Apache License, Version 2. You can clone and freely use these frameworks in your
+projects, submit pull requests, and access the latest documentation at GitHub:
+
+ * https://github.com/SMARTRACTECHNOLOGY/smartcosmos-framework
+
+ * https://github.com/SMARTRACTECHNOLOGY/smartcosmos-devkit
+(where this document was downloaded from)
+
+IMPORTANT: The core of the Objects server itself is presently proprietary and must
```

Comment:
Isn't this the first (or at least one of the first) occurrences of "Objects"? Since we usually refer to _SMART COSMOS_ this might require some additional explanation.

Apart from that, I wouldn't refer to a "Objects server" because it's rather a cluster than a single server.